CAMEL-19892: openapi-rest-dsl-generator - Use jakarta instead of javax

wuersch opened the pull request:
https://issues.apache.org/jira/browse/CAMEL-19892

davsclaus commented:
The CI server says test failures, did you test the code change locally ?

davsclaus commented:
Can you see if this also fails for your locally and more fixes is needed?

[INFO] Camel :: Maven Plugins :: OpenApi REST DSL Generator Maven Plugin FAILURE [02:12 min]

wuersch commented:
The regular tests where fine, the integration tests less so. However, the ITs did not run on my machine in the first place, because the compiler source and target properties in the poms of the ITs were missing. I fixed that now and added the Jakarta dependencies as well. Sorry for the inconvenience!
